examples/chef/common/chef-rvc-mode-delegate.cpp

```diff
@@ -106,17 +108,61 @@ void RvcRunMode::Shutdown()
     }
 }
 
+chip::Protocols::InteractionModel::Status chefRvcRunModeWriteCallback(chip::EndpointId endpointId, chip::ClusterId clusterId,
+                                                                      const EmberAfAttributeMetadata * attributeMetadata,
+                                                                      uint8_t * buffer)
+{
+    chip::Protocols::InteractionModel::Status ret = Protocols::InteractionModel::Status::Success;
+
+    VerifyOrDie(endpointId == 1); // this cluster is only enabled for endpoint 1.
```

Comment:
is verifyOrDier correct or should we just return some `Status::Failure` instead?

Reply:
Right, makes sense.